contracts/Market.sol

```diff
@@ -76,14 +76,17 @@ contract Market is IMarket, Ownable, ERC721 {
 		_margin = margin;
 		_oracle = IOracle(oracle);
 		_signers[owner()] = true;
-
 		timeout = timeoutDays * 1 days;
 	}
 
 	function tokenURI(uint256 tokenId) public view virtual override returns (string memory) {
         return string(abi.encodePacked(baseURI, Strings.toHexString(uint256(uint160(_self)), 20), "/", tokenId.toString()));
     }
 
+	function getOwner() external view returns (address) {
```

Comment:
This is redundant as `owner()` is already a public function in `Ownable`

```    function owner() public view virtual returns (address) {
        return _owner;
    }
```

Reply:
+1, just call `owner()`

Reply:
Owner is not on the IVault interface.  Needs to be wrapped.